azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -780,4 +789,27 @@ private Text getMRTokenRenewerInternal(final JobConf jobConf) throws IOException
         hsProxy.getConnectAddress());
   }
 
+  // we use this function to create retrying metastore client
+  private IMetaStoreClient createMetaStoreClient(HiveConf hiveConf) throws MetaException {
+    // the hookLoader object is not used in this function, but we will keep it for potential storage handler settings.
```

Comment:
The comment is misleading; what you could say is "Custom hook-loader to return a HiveMetaHook if the table is configured with a custom storage handler"